script/cmtool.sh

```diff
@@ -78,6 +79,23 @@ install_puppet()
     fi
 }
 
+install_puppet_collections()
+{
+    echo "==> Installing Puppet Collections ${CM_PC_VERSION}"
+    REDHAT_MAJOR_VERSION=$(egrep -Eo 'release ([0-9][0-9.]*)' /etc/redhat-release | cut -f2 -d' ' | cut -f1 -d.)
+
+    echo "==> Installing Puppet Labs repositories"
+    rpm -ipv "http://yum.puppetlabs.com/puppetlabs-release-${CM_PC_VERSION}-el-${REDHAT_MAJOR_VERSION}.noarch.rpm"
```

Comment:
Use `https` here, as used in the official guide, [puppetlabs_package_repositories.html](https://docs.puppetlabs.com/guides/puppetlabs_package_repositories.html).